packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -364,13 +391,15 @@ export class FeatureDevController {
                 followUps: this.getFollowUpOptions(session?.state.phase),
                 tabID: tabID,
             })
-            this.messenger.sendUpdatePlaceholder(tabID, 'Select an option above to proceed')
         } finally {
             // Finish processing the event
             this.messenger.sendAsyncEventProgress(tabID, false, undefined)
-
-            // Lock the chat input until they explicitly click one of the follow ups
-            this.messenger.sendChatInputEnabled(tabID, false)
+            this.messenger.sendChatInputEnabled(tabID, enableInput)
+            if (enableInput) {
+                this.messenger.sendUpdatePlaceholder(tabID, 'How can this plan be improved?')
```

Comment:
question: has this message been approved by johanna?

I think we need to synchronize to understand what message can be a follow up for retryable errors.